tests/bindings/python/test_component.py

```diff
@@ -22,10 +22,10 @@ def test_inheritance(self):
         from libcellml import Component
 
         x = Component()
-        self.assertIsInstance(x, libcellml.ComponentEntity)
-        self.assertIsInstance(x, libcellml.ImportedEntity)
-        self.assertIsInstance(x, libcellml.NamedEntity)
-        self.assertIsInstance(x, libcellml.Entity)
+        self.assertIsInstance(x, libcellml.componententity.ComponentEntity)
+        self.assertIsInstance(x, libcellml.importedentity.ImportedEntity)
+        self.assertIsInstance(x, libcellml.namedentity.NamedEntity)
+        self.assertIsInstance(x, libcellml.entity.Entity)
```

Comment:
What is the rationale behind using a longer form of classes? I find it less convenient...

Reply:
This is because they are no longer available from the package init file.  This has been done because they are not classes that users should use or interact with in Python.

Reply:
Oh, ok, fair enough.